### ISSUE TITLE
[Accessibility]: Form control element <button> has no associated label

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2191]
## Detailed description

> Describe in detail the issue you're having.

Inside `<ibm-tooltip-definition>`, there is no aria-label for `<button>` element 

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-tooltip-definition>`

> What did you expect to happen? What happened instead? What would you like to see changed?

Each form control must have an associated label

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 3.3.2 Labels or Instructions WCAG technique H44 WCAG technique ARIA16

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178183216-a8544baf-b120-4fb3-94b2-16d5fc6c002d.png)

![image](https://user-images.githubusercontent.com/108860673/178187546-9d94e196-912e-4ea0-9622-6b7ed20a0003.png)

**Comment by Akshat55** [comment]
You're using icons with tooltip definition component which is designed to work with text, instead use the `ibm-tooltip-icon` or `ibm-tooltip` components.
